en/docs/security-announcements/cve-justifications/2025/BDSA-2022-1920.md

```
@@ -0,0 +1,37 @@
+
+Additionally, WSO2 will address this outdated dependency by upgrading it to a later version in the future.
+
+### CONCLUSION
+
+- WSO2 API Manager is not impacted by this vulnerability.
+- No customer action is required at this time.
+
+### REFERENCES
+[^1]: [https://openhub.net/vulnerabilities/bdsa/bdsa-2022-1920](https://openhub.net/vulnerabilities/bdsa/bdsa-2022-1920)
+[^2]: [https://issues.apache.org/jira/browse/PDFBOX-5401](https://issues.apache.org/jira/browse/PDFBOX-5401)
+[^3]: [https://github.com/apache/pdfbox/commit/0058e8c970cfe21254144343a3dcd9fd2ca96b70](https://github.com/apache/pdfbox/commit/0058e8c970cfe21254144343a3dcd9fd2ca96b70
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)